android/app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -16,77 +16,84 @@
 
     @PrimaryKey
     private Integer id;
-    @JsonProperty("event_url")
     private String eventUrl;
     private String description;
-    @JsonProperty("licence_details")
     private LicenseDetails licenseDetails;
-    @JsonProperty("start_time")
     private String startTime;
-    @JsonProperty("end_time")
     private String endTime;
-    @JsonProperty("ticket_url")
     private String ticketUrl;
     private String topic;
     private String thumbnail;
     private String timezone;
     private String logo;
-    @JsonProperty("searchable_location_name")
     private String searchableLocationName;
     private String large;
-    @JsonProperty("background_image")
     private String backgroundImage;
-    @JsonProperty("call_for_papers")
     private CallForPapers callForPapers;
-    @JsonProperty("location_name")
     private String locationName;
     private String name;
     private Copyright copyright;
     private String privacy;
-    @JsonProperty("placeholder_url")
     private String placeholderUrl;
-    @JsonProperty("social_links")
     private RealmList<SocialLink> socialLinks;
     private Double longitude;
-    @JsonProperty("organizer_name")
     private String organizerName;
-    @JsonProperty("schedule_published_on")
     private String schedulePublishedOn;
     private String state;
     private Version version;
-    @JsonProperty("has_session_speakers")
     private Boolean hasSessionSpeakers;
-    @JsonProperty("sub_topic")
     private String subTopic;
     private Double latitude;
-    @JsonProperty("organizer_description")
     private String organizerDescription;
     private String identifier;
     private String type;
     private String email;
-    @JsonProperty("code_of_conduct")
     private String codeOfConduct;
 
+    @JsonGetter("event_url")
```

Comment:
but if the atttribute has both a setter and a getter than should i make the json property the old one which will be used for the getter and update the setter with multiple field names?